### ISSUE TITLE
Add build agent metrics

### ISSUE THREAD

**Issue by kersson** [buildkite/feedback#182]
It would be great to be able to view/track agent metrics (e.g. performance or reliability).

As a first step, it would be great to just list build history by agent (e.g. on the agent's info page).

**Comment by jgavris** [comment]
👍

**Comment by sschaetz** [comment]
👍

**Comment by apparentlymart** [comment]
Agreed on this. My main use-case is having a tally of how many builds have failed vs. succeeded recently for a given agent, so that I can quickly understand when one of our agents has failed in some way and fix it.

Right now it's tricky to correlate a set of errant failures spread over multiple pipelines to prove that they all came from a single agent. Ideally I'd like to see a summary of recent failure count (say, in the last day?) on the main agent listing, so it's easy to spot the ones that are having problems while skimming the list. However, I'd also be happy enough with the original suggestion of a history on each agent's detail page.

**Comment by keithpitt** [comment]
Thanks for the feedback everyone! This is a good idea(tm)

We've already started a project internally to push out a version 2 of the agents page. This should hopefully include some of this information.

We're working hard on pushing out version 2 of our Pipelines List, that you should see drop any day now!

**Comment by sschaetz** [comment]
Any more progress on this beyond "agents page version 2"?

**Comment by rjschnei** [comment]
This feature would be incredibly valuable... is there an ETA?
